experiments/AMIP/modular/Project.toml

```diff
@@ -48,7 +48,49 @@ Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
-ClimaTimeSteppers = "0.6.0"
+ArgParse = "1.1"
```

Comment:
```suggestion
ArgParse = "1"
```

Reply:
Hi @charleskawczynski , thanks. As I mentioned in the PR description, or versions > 1.0, I used the same as the ClimaAtmos.jl/examples environment (keeping the minors), as a reference and to make sure they are compatible. See [`examples/Project.toml`](https://github.com/CliMA/ClimaAtmos.jl/blob/main/examples/Project.toml). What should we do?

Reply:
I'm updating ClimaAtmos' compat entries in https://github.com/CliMA/ClimaAtmos.jl/pull/1603, so let's just apply my suggestion